### ISSUE TITLE
Request: After card number entered, show 'dots' to denote hidden numbers

### ISSUE THREAD

**Issue by mhousser** [dbachelder/CreditCardEntry#31]
After you enter the 16 numbers, only 4 are shown, with the first 12 being hidden. Very cool!

However, I think it would be a bit more obvious to users if the first 12 digits were replaced with 'dots' (or some such symbol) to make it _really_ obvious that the card was entered successfully and that we're only hiding numbers from them as a security measure.

Would something like this be possible? I've seen it elsewhere and it felt more user friendly. Perhaps configurable with a true/false attribute?

**Comment by dbachelder** [comment]
That field needs to slide over to make room for more data entry. On smaller devices it wouldn't be possible to show both the full 16 characters and the additional input fields. 

Last-4 is a very well known paradigm for obscuring credit cards... but if you want to mock up an alternative and test the viability a bit.. I'm all ears.

Please re-open if you feel this deserves further consideration.

**Comment by mhousser** [comment]
Ah ok. Well, I was thinking more like - whatever space that _is_ available be backgrounded with some sort of 'dots' motif, instead of actually trying to preserve the full width of the 12 characters that used to be there.

Anyway, it was just a picky request. I'm not going to be annoying and re-open. =P Nice library!

**Comment by dbachelder** [comment]
I think I get what you mean... I'll reopen for future consideration.

**Comment by mhousser** [comment]
Behold, the greatest mock-up ever produced!

I guess I had more of a 'repeating background' in mind, to make up for the fact that we can't predict the width of the view as you mentioned.

There is also the opportunity to have different approaches depending on the width. For example, the Facebook LoginButton control shows the text 'Log in with Facebook' if the button is wide enough to display it, and only 'Log in' if the button isn't wide enough (it does a text width calculation to check first).

![screenshot_2015-09-16-13-17-35](https://cloud.githubusercontent.com/assets/832078/9917684/dde68794-5c77-11e5-8c3d-211fc449e4c2.png)

**Comment by dbachelder** [comment]
Yeah.. this is certainly something we can look at.. thanks for the mock. I'll give it some thought.
